app/javascript/spotlight/channels/index.js

```diff
@@ -0,0 +1 @@
+// Import all the channels to be used by Action Cable
```

Comment:
Is this file needed?

Reply:
It is automatically created by ActionCable. I wasn't sure if it was something we would want to keep.

Reply:
Automatically created into an engine? How does that work?

Reply:
In a site. I create the site ran the auto creation task and moved everything.

Reply:
```
(!) Unresolved dependencies
https://rollupjs.org/troubleshooting/#warning-treating-module-as-external-dependency
@rails/actioncable (imported by "app/javascript/spotlight/channels/consumer.js")
created app/assets/javascripts/spotlight/spotlight.esm.js in 184ms
```

Reply:
You don't want to move these files into the engine.  They should be generated by the auto creation task in the host app.